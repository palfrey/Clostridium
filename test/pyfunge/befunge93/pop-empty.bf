#v$"gnihton seod kcats ytpme htiw $ :DOOG",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@
 <@,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"BAD: $ with empty stack reflects"
