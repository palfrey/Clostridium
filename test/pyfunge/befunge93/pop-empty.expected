GOOD: $ with empty stack does nothing
